.env.example

```
@@ -200,3 +200,12 @@ DAILY_SUMMARY_HOUR = "21" # Hour of the day (0-23) to send daily summary alerts
 OUTBOUND_RATE_LIMIT_MAX_CALLS=7
 OUTBOUND_RATE_LIMIT_WINDOW_SECONDS=3600
 # OUTBOUND_RATE_LIMIT_BLOCK_ENABLED is a dynamic config key (set via Redis, default: false)
+
+# Blueprint (template creation agent) — Vertex-backed Claude
+# Required for the LLM-driven planner / question renderer / extractor.
+# Without these, Blueprint falls back to a deterministic no-LLM walk
+# useful only for dev/CI.
+BLUE_PRINT_GOOGLE_CREDENTIALS_JSON=  # Service-account JSON (required)
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)